### ISSUE TITLE
Code - Mobile banner takeover

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#8829]
In #8821 we'll graduate a CRO experiment that puts a banner at the top of /new for mobile visitors to this page. #8823 has designs.

## success criteria ##

- [x] Use Bedrock's banner container if possible -- if not possible for some reason, make sure this content is wrapped in a [data-nosnippet](https://developers.google.com/search/reference/robots_meta_tag#data-nosnippet-attr)
- [x] new banner is opened to localization
- [x] integrate with new Fluent pipeline release timeline

**Comment by alexgibson** [comment]
@cgonz0 @hoosteeno 

This is now up on demo for initial review: https://www-demo1.allizom.org/en-US/firefox/new/

To make preview easier (you can only see the banners on demo using an Android / iOS device), I attached screenshots below. Note the design file didn't feature a mockup for wider viewports (e.g. tablets), so I just did what I thought felt right.

For the badges I used adjust links (assuming this is what we want here).

**Android (small)**

![image.png](https://images.zenhubusercontent.com/5c927d6baa8dca0a81399557/a9fecb96-cfd3-4e9b-b39a-c5ab0dde4c95)

**Android (wide)**

![image.png](https://images.zenhubusercontent.com/5c927d6baa8dca0a81399557/eaa4e0ec-68ed-4372-bee5-fbe5474f3d57)

**iOS (small)**

![image.png](https://images.zenhubusercontent.com/5c927d6baa8dca0a81399557/67f2a0ee-ef9a-4500-9aab-06b92c8f1688)

**iOS (wide)**

![image.png](https://images.zenhubusercontent.com/5c927d6baa8dca0a81399557/4351d1c5-19e5-4235-a0e9-9a185962eff9)

**Comment by hoosteeno** [comment]
@cgonz0 if the wide treatment works for you then I am r+ on this. Thank you.

@alexgibson will we hold this until Fluent is in place?

**Comment by alexgibson** [comment]
> @alexgibson will we hold this until Fluent is in place?

That's my plan, yes.

**Comment by cgonz0** [comment]
@alexgibson thanks for these. these tablets are getting larger:) given that, a few small suggestions here... 
could we bump down the button to 40px height? feels a little large in the smallest width.
for the large tablet width, i'm good with the background repeating but could we stretch the full text to take up one line and bump down the size to 32px. this way we can just have the text flow right into the button and it'll feel a little more cohesive.
let me know if you wanna chat about it
thanks

**Comment by alexgibson** [comment]
>  these tablets are getting larger:)

Yeah, on a large iPad pro in landscape orientation the browser width is 1366px wide, which is not really any different to many desktop screens. It's no longer really possible to differentiate devices based on screen size alone.

> could we bump down the button to 40px height?

The current dimensions of 152px x 45px is the standard we use across the site for all app store buttons, so I can't easily start tweaking this here without it effecting layout elsewhere. If we want to adjust this then I'd suggest we start doing it in a consistent way on all pages. Probably worth filing a separate issue for.

> i'm good with the background repeating but could we stretch the full text to take up one line and bump down the size to 32px. this way we can just have the text flow right into the button and it'll feel a little more cohesive.

I think this makes sense yeah. It does start to deviate from our existing banner design / layout, but seeing as we didn't factor this in originally I'll see what I can do here.

**Comment by alexgibson** [comment]
@cgonz0 this is now updated on demo. I've taken a screenshot to help preview. Any further tweaks just let me know, thanks!

![image.png](https://images.zenhubusercontent.com/5c927d6baa8dca0a81399557/f8f5293b-68bd-4421-b6f2-7e50c68738ce)

**Comment by cgonz0** [comment]
> The current dimensions of 152px x 45px is the standard we use across the site for all app store buttons, so I can't easily start tweaking this here without it effecting layout elsewhere. If we want to adjust this then I'd suggest we start doing it in a consistent way on all pages. Probably worth filing a separate issue for?

didn't know this was our standard - so, let's leave it. i'll make a note for future designs.

> I think this makes sense yeah. It does start to deviate from our existing banner design / layout, but seeing as we didn't factor this in originally I'll see what I can do here.

i also suggested leaving the same layout (centered text with button beneath) if we don't wanna deviate too much. the height of the banner will be wider but i think the layout still works.
but, given that you've already tweaked the layout into a single row, let's go with that one.
**let's drop the text to 32px** 
otherwise, it looks good to me, thanks!

**Comment by alexgibson** [comment]
Updated, thanks!

**Comment by alexgibson** [comment]
This is now pretty much code complete, but is now blocked on https://github.com/mozilla/bedrock/pull/8803 which will migrate /firefox/new/ to Fluent. Once that is merged later this week I'll pick this back up to make it ready for L10n.

**Comment by alexgibson** [comment]
This is now unblocked and ready for code review.
